mindmeld/system_entity_recognizer.py

```diff
@@ -22,6 +22,7 @@
 )
 
 NO_RESPONSE_CODE = -1
+SYS_ENTITY_REQUEST_TIMEOUT = float(os.environ.get("MM_SYS_ENTITY_REQUEST_TIMEOUT", 1.0))
```

Comment:
We should probably enforce a > 0.0 value?

Reply:
For reference, here's what happens when you throw a negative timeout into requests
`ValueError: Attempted to set connect timeout to -1.0, but the timeout cannot be set to a value less than or equal to 0.
`
It's better to warn the user if we see an error rather than defer it to later when we call duckling.

Reply:
Addressed this issue.

Reply:
If the environment variable doesn't parse as a float, do we want to catch the resulting `ValueError` and reraise it as a `MindMeldError`?

Reply:
Probably we need to do some validation here.

Reply:
resolved